### ISSUE TITLE
Provided Lego separators are incomplete

### ISSUE THREAD

**Issue by alexd2580** [ryanoasis/powerline-extra-symbols#8]
With the provided lego separators `E0CE` and `E0D2`, `E0D4` it is impossible to construct the prompt shown in the fourth line:

https://github.com/ryanoasis/powerline-extra-symbols/blob/master/preview.png

**Comment by ryanoasis** [comment]
I see what you are saying and while I don't remember exactly how it ended up appearing like that you can get very close with those symbols:

```sh
echo -e '\e[38;5;250m\e[48;5;240m ryan \e[48;5;31m\e[38;5;240m\e[48;5;240m\uE0CE\e[38;5;15m\e[48;5;31m  ~\e[48;5;236m\e[38;5;31m\e[48;5;31m\uE0CE\e[38;5;15m\e[48;5;236m  $\e[0m\e[38;5;236m\e[48;5;236m\uE0CE\e[0m'
```

![selection_030](https://user-images.githubusercontent.com/8083459/52895375-89937880-3187-11e9-90a3-7df7d39de34a.png)

**Comment by alexd2580** [comment]
Yeah, close, but not quite =( unfortunately the vertical line  is rather noticeable.
